### DISCUSSION TITLE
Designing carbon v5.+ with Angular 16

### DISCUSSION THREAD

**Discussion by ai-mine-max** [IBM/carbon-components-angular#2693] [Q&A]
Hi there. I just started using carbon for a multi-module project. It seems the examples provided for the tutorials are mainly for Angular 8-11. Is there example for carbon v5 with angular 16?

**Comment by Akshat55** [comment]
Sorry for the late response, but to answer the question there is the [carbon-angular-starter](https://github.com/carbon-design-system/carbon-angular-starter). 

The starter is using CCA@v5 angular Angular 16.
